### ISSUE TITLE
Focus indicator styles

### ISSUE THREAD

**Issue by aviannamiller** [omeka/omeka-s#2316]
Freedom, Lively, and Thanks, Roy override system focus outlines. [Higher-level guidance from WCAG 2.2](https://www.w3.org/WAI/WCAG22/Understanding/focus-appearance) dictates that focus indicators, if overriding user agent styles, should have at least 2px perimeters. Currently, these themes utilize thin outlines that fall short of this guidance, and in some cases, have pretty low visibility. Take this example from Freedom's menu drawer, where focus is on the "Browse" link:

<img width="643" alt="Open menu drawer. A thin focus indicator intersects bounding boxes and overlaps lines between list items." src="https://github.com/user-attachments/assets/26570fec-d573-4f87-9bcc-c87927323e42" />

**Comment by nelsonamaya82** [comment]
Custom focus styles were removed in Freedom and Lively. Now they use default user agent styles.

**Comment by aviannamiller** [comment]
@nelsonamaya82–Great, thank you for the changes.

@kimisgold–Leaving this open for Thanks, Roy.

**Comment by aviannamiller** [comment]
Actually, @nelsonamaya82 the focus indicator becomes a bit lost on "Previous" links within the mobile menu (both themes). Is there any good way around that?

**Comment by kimisgold** [comment]
Custom focus has been removed in Thanks, Roy.

**Comment by aviannamiller** [comment]
> Custom focus has been removed in Thanks, Roy.

Looks great, thanks!

**Comment by nelsonamaya82** [comment]
@aviannamiller I improved the focus indicator visibility on "Previous" links within the mobile menu on both themes.

**Comment by aviannamiller** [comment]
@nelsonamaya82 Excellent, thank you!
